utils/hwinfo/Makefile

```diff
@@ -0,0 +1,87 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hwinfo
+PKG_VERSION:=21.70
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/openSUSE/hwinfo/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=bc3c4a4498e4abc32a54497ced715bbae5dfd19dd999da294bca6d69fea2db52
+
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=Alberto Bursi <bobafetthotmail@gmail.com>
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:= hwinfo/host e2fsprogs/host
```

Comment:
e2fsprogs/host is not needed here.

Why is it needed at all? there's tools/e2fsprogs.

Reply:
hmm, yeah it seems I got confused, the uuid.h header I need comes from the tools/e2fsprogs https://github.com/openwrt/openwrt/blob/master/tools/e2fsprogs/Makefile

Reply:
ok removed all e2fsprogs/host and recompiled, it's ok without it